Packs/Rapid7InsightVMCloud/Integrations/Rapid7InsightVMCloud/Rapid7InsightVMCloud.yml

```diff
@@ -0,0 +1,107 @@
+category: Vulnerability Management
+commonfields:
+  id: Rapid7 InsightVM Cloud
+  version: -1
+configuration:
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- defaultvalue: "false"
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- defaultvalue: https://us.api.insight.rapid7.com/vm
+  display: Server URL (e.g., https://example.net)
+  name: base_url
+  required: true
+  type: 0
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: InsightVM is a Vulnerability Management Tool which Scan your Network, Eliminate Vulnerabilities, Track and Communicate progress.
+display: Rapid7 InsightVM Cloud
+name: Rapid7 InsightVM Cloud
+script:
+  commands:
+  - arguments:
+    - description: The identifier of the asset to retrieve the details for.
+      name: asset_id
+      required: true
+    description: Returns the assessment and details of an asset (specified by id). Only assets which the caller has access to can be returned.
```

Comment:
Edited:
Returns the assessment and details of an asset (specified by ID). Only assets that the caller has access to can be returned.

Reply:
I have edited the suggested change.